packages/framework/tests/Test/Provider/TestCurrencyProvider.php

```diff
@@ -2,7 +2,7 @@
 
 declare(strict_types=1);
 
-namespace Tests\App\Functional\Model\Pricing\Currency;
+namespace Tests\FrameworkBundle\Test\Provider;
```

Comment:
This change should be mentioned in UPGRADE, right?

Reply:
Added, thanks